blog/2015/10/python-debriefing-summary.html

```diff
@@ -0,0 +1,77 @@
+  We now have a summary of the feedback we've received in our bi-weekly debriefing meetings.
+  The recurring themes are:
+</p>
+<ol>
+  <li><p>A greater choice of exercises and multiple choice questions would allow instructors to select domain specific examples and cater to varying levels of learner.</p></li>
+  <li><p>Some instructors added an explanation of the IPython/Jupyter/SpyderIDE environment.</p></li>
+  <li><p>Some instructors added an explanation of basic Python datatypes before presenting the lessons.</p></li>
+  <li><p>Comments that there is too much material to fit into a workshop and that some sections seem rushed.</p></li>
+  <li><p>Request for a better explanation of the advantages of the Anaconda Distribution at the start of workshops and resources for post workshop learning.</p></li>
+  <li><p>Comments on presenters style were positive for funny and entertaining examples and negative for highly mathematical examples.</p></li>
+</ol>
+<!-- end excerpt -->
+<p>The details are listed below&mdash;we plan to use all of this when we redesign our basic Python lesson.</p>
+<ul>
+  <li>Departures
+    <ul>
+      <li><p>Changed lesson to be more relevant to audience +1</p></li>
+      <li><p>Instructor used own examples to change emphasis of lesson to loops</p></li>
+      <li><p>Re-organised the flow of lesson</p></li>
+    </ul>
+  </li>
+  <li>Additions
+    <ul>
+      <li><p>Explained differences between bash, python in the shell, IPython notebook</p></li>
+      <li><p>Short Jupyter Notebook introduction</p></li>
+      <li><p>Demonstrated Spyder IDE as well as IPython</p></li>
+      <li><p>Basic data types explained before starting +1</p></li>
+    </ul>
+  </li>
+  <li>Requests
+    <ul>
+      <li><p>More exercise options in repo +1</p></li>
+      <li><p>More multiple choice questions</p></li>
+      <li><p>Directions to post workshop resources needed</p></li>
+      <li><p>Explain that Anaconda will install in a separate folder and not affect existing Python</p></li>
+    </ul>
+  </li>
+  <li>Comments
+    <ul>
+      <li><p>Testing section seemed rushed</p></li>
+      <li><p>lesson material too long to fit workshop length +1</p></li>
+    </ul>
+  </li>
+</ul>
+<p><strong>IPython Notebook</strong></p>
+<ul>
+  <li><p>Using Python Notebook with Git is challenging.</p></li>
+  <li><p>Lack of extensive history in notebook (compared to shell) hinders learners +1</p></li>
+  <li><p>Starting with an empty notebook better than using template code</p></li>
+  <li><p>Notebooks should be saved to repo in a cleared state, split up monolithic Jupyter Notebook (i.e. clear output) +1</p>
+    <ul>
+      <li>
+	<p>implies not live coding.</p>
+      </li>
+    </ul>
+  </li>
+</ul>
+<p><strong>Instructor Style</strong></p>
+<ul>
+  <li><p>Complaint - prime numbers example too mathematical</p></li>
+  <li><p>Complaint - Not enough discussion prior to challenges. learners - &quot;I don't know where to start!&quot;</p></li>
```

Comment:
Reword to

~~~
  <li><p>Complaint - Not enough discussion prior to challenges, e.g. learners said &quot;I don't know where to start!&quot;</p></li> 
~~~